.github/ISSUE_TEMPLATE/test-failures.yml

```diff
@@ -0,0 +1,59 @@
+name: 📑 Test failure report
+description: Issue Template for Reporting Testing Failures.
+title: "[Test Case ID]: <DESCRIPTIVE TITLE>"
+labels: ["release testing"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Test failure report
+        Describe the failure you encountered while testing a release.
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Describe what was expected to happen during testing.
+      placeholder:
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: Describe what actually happened during testing, including any errors or failures encountered.
+      placeholder:
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Provide detailed steps to reproduce the testing failure.
+      placeholder:
+    validations:
+      required: true
+  - type: input
+    id: release
+    attributes:
+      label: Release
+      description: Include the name of the release in which the testing was performed.
+      placeholder: Ironwood
+    validations:
+      required: true
+  - type: textarea
+    id: more-info
+    attributes:
+      label: Additional information
+      description: Include any additional information that may be helpful in understanding the testing failure, such as screenshots or log files.
+  - type: dropdown
+    id: urgent
+    attributes:
+      label: Failure impact
+      description: How badly impacts the release this failure?
+      options:
+        - Low (Not a release blocker, nice to have)
+        - Medium (Needs to be fixed but not a release blocker)
+        - High (It's a release blocker)
```

Comment:
How do I know if it's a release blocker?

Reply:
That's a great question. Maybe we should 1st open that discussion, document the decision and then add this prompt. I opened an issue to track it: https://github.com/openedx/wg-build-test-release/issues/292